### ISSUE TITLE
BPARules: Date rules should be ISO 8601 compliant

### ISSUE THREAD

**Issue by bgarcevic** [microsoft/Analysis-Services#181]
The date formatting rules changes the dates to mm/dd/yyyy whereas which is only used in the US. All other countries use either dd/mm/yyyy or ISO 8601 yyyy-mm-dd or some variation thereof.

**Comment by m-kovalsky** [comment]
This was brought up in Issue #135. If you want this to be of another date format, I recommend updating your rules file (the fix expression) to use the date format you desire.

**Comment by bgarcevic** [comment]
If I made a danish translation of the project, to this repo, could I change the date format?

**Comment by m-kovalsky** [comment]
I do not believe that a Danish translation is necessary. There are 2 options:

1. Updating the fix expression for this rule within the BPARules.json file after you download it.
2. Running the following C# script in Tabular Editor (changing the parameter in the first line as desired).

```c#
string dateFormat = "dd/mm/yyyy"; // Enter the desired date format 

foreach (var c in Model.AllColumns.Where(a => a.DataType == DataType.DateTime && a.Name.IndexOf("Date", StringComparison.OrdinalIgnoreCase) >= 0))
{
    c.FormatString = dateFormat;
}
```
